Chinese/迪力.txt

```diff
@@ -0,0 +1 @@
+迪力会成为一名伟大的程序员，他正在为这个梦想不断奋斗着。继续勇往直前吧，Dilyar!
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.